### ISSUE TITLE
train classes mis-leading..

### ISSUE THREAD

**Issue by sycophant-stone** [zllrunning/face-parsing.PyTorch#19]
Since there are 17 classes[1] in the prepare stags, while there are 19 classes in pretrained model from `test.py`.[2]
So,my concern is what is the differences between those two models?

[1]: https://github.com/zllrunning/face-parsing.PyTorch/blob/master/prepropess_data.py#L17-L18
[2]: https://github.com/zllrunning/face-parsing.PyTorch/blob/master/test.py#L56

**Comment by showkeyjar** [comment]
they said that it contains last background part, but I test result found the class name make more confused
![parsing_torch](https://user-images.githubusercontent.com/16154787/136679599-eb6522df-ff9e-4afe-8e0f-adc8d5d6c854.jpg)

and this is my label order: ['skin', 'l_brow', 'r_brow', 'l_eye', 'r_eye', 'eye_g', 'l_ear', 'r_ear', 'ear_r',
            'nose', 'mouth', 'u_lip', 'l_lip', 'neck', 'neck_l', 'cloth', 'hair', 'hat', 'bg']

**Comment by Ir1d** [comment]
https://github.com/switchablenorms/CelebAMask-HQ/tree/master/face_parsing#well-trained-model

**Comment by mohamad-hasan-sohan-ajini** [comment]
> 

Background is first class. I tested [this model](https://drive.google.com/open?id=154JgKpzCPW82qINcVieuPH3fZ2e0P812) (the one that is mentioned in README.md) and found this labeling:

['bg', 'skin', 'l_brow', 'r_brow', 'l_eye', 'r_eye', 'eye_g', 'l_ear', 'r_ear', 'ear_r',
'nose', 'mouth', 'u_lip', 'l_lip', 'neck', 'neck_l', 'cloth', 'hair', 'hat']

**Comment by fortunechen** [comment]
> https://github.com/switchablenorms/CelebAMask-HQ/tree/master/face_parsing#well-trained-model

This label is not suitable for this model
